src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -493,6 +495,7 @@ private void setup(OptionSet options, Properties properties) {
 		this.replayMode =     fetchBooleanOption("replay", options, null, false);
 		this.masterRecovery = fetchBooleanOption("master_recovery", options, properties, false);
 		this.ignoreProducerError = fetchBooleanOption("ignore_producer_error", options, properties, true);
+		this.recaptureSchema = fetchBooleanOption("recapture_schema", options, properties, false);
```

Comment:
no, I mean here:  passing `properties` allows this into the config.properties here.

Reply:
👍 Good point!